src/device-registry/stage-docker-push.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+echo -e "\n ==== 🚚  🚁 Pushing API docker image 🛫   ==== \n"
+echo -e "\nHope you have built airqo-stage-device-registry-kafka-connection:latest\n"
+
+# docker push airqo-stage-device-registry-api:latest
+
+echo "💪 Push complete"
+echo "💫 Image : airqo-stage-device-registry-api:latest"
+
+echo -e "\n ==== 🚚  🚁 Pushing Kafka connections docker image 🛫   ==== \n"
+echo -e "\nHope you have built airqo-stage-device-registry-kafka-connection:latest\n"
+
+# docker push airqo-stage-device-registry-kafka-connection:latest
+
+echo "💪 Push complete"
+echo -e "💫 Image : airqo-stage-device-registry-kafka-connection:latest \n"
```

Comment:
The image name specified the k8s yaml file is `airqo-stage-device-registry-kafka:latest`. @NoahNsimbe

Reply:
👍, I have made the changes